i18n/en.toml

```diff
@@ -149,6 +152,9 @@ other = "Window Start"
 [certificate_transparency_window_end]
 other = "Window End"
 
+[certification_practices_framework]
+other = "Certification Practices Framework"
```

Comment:
This should be "Certification Practices Statement", in the text and the label.

Reply:
Wow I don't know how I did that, fixed!